html/pfappserver/lib/pfappserver/PacketFence/Controller/Graph.pm

```diff
@@ -296,7 +296,7 @@ sub _buildGraphiteURL :Private {
 
     my $options =
       {
-       graphite_host => $management_ip,
+       graphite_host => $Config{'general'}{'hostname'} . '.' .  $Config{'general'}{'domain'},
```

Comment:
If the dns entry is not set correctly then it will not work. There is probably an env variable in catalyst with the host value.